Source/Filters/Video/YPVideoFiltersVC.swift

```diff
@@ -229,7 +230,7 @@ open class YPVideoFiltersVC: UIViewController, IsMediaFilterVC {
     }
 
     private func setupRightBarButtonItem() {
-        let rightBarButtonTitle = isFromSelectionVC ? YPConfig.wordings.done : YPConfig.wordings.next
+        let rightBarButtonTitle = shouldShowDone || isFromSelectionVC ? YPConfig.wordings.done : YPConfig.wordings.next // Test 3
```

Comment:
We probably don't need the `// Test 3` comment anymore?

Reply:
Yep, didn't push up that commit. Pushing it now. Thanks!